lib/vmdb/appliance.rb

```diff
@@ -87,10 +87,11 @@ def self.log_server_identity
       # the file must not be named ".log" or it will be removed by logrotate, and it must contain the Server GUID (by which the appliance is known in the vmdb,
       # the build identifier of the appliance as it is being started,  the appliance hostname and the name of the appliance as configured from our configuration screen.
 
-      startup_fname = File.join(Rails.root, "log/last_startup.txt")
-      FileUtils.rm_f(startup_fname) if File.exist?(startup_fname)
+      startup_fname = Rails.root.join("log", "last_startup.txt")
+      startup_fname.delete if startup_fname.exist?
```

Comment:
Small nitpick... maybe for a followup... this is no longer a `startup_fname`.  Maybe `last_startup_file` makes sense.

Reply:
Done